src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3027,20 +3027,29 @@
       </PropertyComment>
     </Field>
   </AbstractNode>
-  <Node Name="NamespaceDeclarationSyntax" Base="MemberDeclarationSyntax">
+    <AbstractNode Name="BaseNamespaceDeclarationSyntax" Base="MemberDeclarationSyntax">
```

Comment:
><AbstractNode [](start = 4, length = 13)

There's an indentation difference with rest of XML file

Reply:
fixed.